agent/bench-scripts/pbench-trafficgen

```diff
@@ -915,6 +915,38 @@ fi
 # At this point all parameter checking has been done.
 ###
 
+# Build up a result directory identifier from parameters
+tg_config="tg:$(echo ${traffic_generator} | sed -e 's/-txrx//')"
+if [ $traffic_generator == "trex-txrx-profile" -a "${traffic_profiles}" != "" ]; then
+	traffic_profiles_list=""
+	for traffic_profile in $(echo ${traffic_profiles} | sed -e s/,/" "/g); do
```

Comment:
Agreed.

I was trying not to make those kinds of changes because they were not directly related to fixing this problem, and those loop control constructs, if borked by an attempt to replace the external use of `sed` with `bash` equivalent, could cause more problems.

I'd rather do that kind of change outside this PR.